api.ts

```diff
@@ -153,9 +153,16 @@ export interface SourceFileConfiguration {
     readonly defines: string[];
 
     /**
-     * The compiler to emulate.
+     * The platform, compiler, and architecture variant to emulate.

```

Comment:
Can we clarify somehow that the enum values without a platform will resolve to the host platform?